src/chrome/content/rules/rockfile.eu.xml

```diff
@@ -28,6 +28,25 @@
 
 	<securecookie host="^\." name="^(?:__cfduid|cf_clearance)$" />
 	<securecookie host="^\w" name="." />
+	
+	<!-- download pages redirect to http -->
+	<exclusion pattern="^https?://(?:www\.)?rockfile\.eu/[A-z0-9]+\.html" />
```

Comment:
We only exclude http domains, https is not touched by the rule. Please also remove the non-capturing group.